modules/es-extensions/publisher/extensions/app/greg-diff/themes/default/js/diff.js

```diff
@@ -51,12 +51,12 @@ $('.container-fluid').resize(function(){
  */
 function setViewPanelsHeight(){
     if($(window).width() > 768){
-        $(viewPanel).css('height', ($('body').height() - appBarHeight));
+        $(viewPanel).css('height', ($('body').height() - ($(window).height()*.1)));
```

Comment:
When we allow code mirror to set the height automatically it looks smaller. '768' is referred to the height component of 1024x768 resolution which is considered to be the basic web designing resolution. Therefore, if we detect a screen height larger than 768px, we are obtaining 90% of the screen height responsively for the code mirror divs. If the screen is smaller than 768, we allow code mirror to set it to the default heights.